lnrpc/lightning.proto

```diff
@@ -3506,7 +3506,30 @@ message PolicyUpdateRequest {
     // If true, min_htlc_msat is applied.
     bool min_htlc_msat_specified = 8;
 }
+
+message FailedUpdate {
+    enum Reason {
+        UNKNOWN = 0;
+        PENDING = 1;
+        INACTIVE = 2;
+        NOTFOUND = 3;
+        INVALID = 4;
+        INVALID_PARAMETER = 5;
+    }
+
+    // The channel that the policy update report belongs to.
+    string channel_point = 1;
```

Comment:
I think that we have a dedicated type for outpoints?

Reply:
We have. Just applied it on the new update. If you think it becomes to bloated I'll change it back to a string.